test/mpi/rma/Makefile.am

```diff
@@ -417,3 +419,9 @@ lockall_dt_flushlocalall_SOURCES = lock_x_dt.c
 
 putpscw1_CPPFLAGS = $(AM_CPPFLAGS)
 putpscw1_SOURCES = putpscw1.c
+
+win_dynamic_acc_collattach_CPPFLAGS = -DUSE_INFO_COLL_ATTACH $(AM_CPPFLAGS)
+win_dynamic_acc_collattach_SOURCES = win_dynamic_acc.c
+
+aint_collattach_CPPFLAGS = -DUSE_INFO_COLL_ATTACH $(AM_CPPFLAGS)
+aint_collattach_SOURCES = aint.c
```

Comment:
Missing `\n`

EDIT: never mind since you gonna overwrite it in the next commit :)